### PR TITLE
fix: handle non-existing current context

### LINKED ISSUE #13: ktx command fails without a current context
### Steps

1. Remove the `current-context` entry from `~/.kube/config`
2. Run `ktx`

### Observation

An error is shown (1):

> error: current-context is not set

### Expectation

The tool opens the available contexts as usual.

### PR DESCRIPTION
**Pull request by cmeury**
fixes #13

Check for exit code, then set `current` to empty string. This will have no effect on the list of contexts, and also appending it at the end works because of the added control character `\c` which skips the newline.

### CODE REVIEW

**Review by kubosuke — APPROVED** [review]

**Comment by cmeury** [comment]
Resolved the conflict, ready to merge.

File: bin/ktx [hunk 1]
```diff
@@ -12,7 +12,13 @@ if [ ! -x "$(which fzf 2>/dev/null)" ]; then
...
 fi
 
-current="$(kubectl config current-context)"
+exit_code=0; kubectl config current-context || exit_code=$?
+if [[ $exit_code -eq 0 ]]; then
+  current="$(kubectl config current-context)"
+else
+  current=""
+fi
+
```
Comment (antonu17) [inline bin/ktx]:
```suggestion
current="$(kubectl config current-context || true)"
```

this has the same effect and spawns one less process

**Review by koenbollen — APPROVED** [review]

**Comment by cmeury** [comment]
Still ready to merge :)

**Comment by koenbollen** [comment]
@joosterman Can you maybe hit that merge button?

**Comment by koenbollen** [comment]
Thanks! 🤗

**Pull request merged**